src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs

```diff
@@ -4,164 +4,186 @@
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using System.Linq;
+using System.Net;
+using System.Text;
 using ONE.Models.CSharp;

```

Comment:
I think the usings on line 9 and 10 could be combined.  I'd probably suggest using Proto. everywhere in this file.

Reply:
Combined the using statements